src/regressiontestoutput.txt

```diff
@@ -0,0 +1,23 @@
+
+FilePath: ../src/activities/testactivity/testactivity.js
+	 ERROR       Replace original GTK VERSION AUTHOR label by your own name
+	 WARNING     Replace original QT Quick copyright label email by your own email address
+
+FilePath: ../src/activities/testactivity/Testactivity.qml
+	 ERROR       Replace original GTK VERSION AUTHOR label by your own name
+	 WARNING     Replace original QT Quick copyright label email by your own email address
+	 WARNING     "testactivity activity" qsTr may not be used
+
+FilePath: ../src/activities/testactivity/ActivityInfo.qml
+	 WARNING     difficulty: 1 may not be updated.
+	 ERROR       author: "Your Name &lt;yy@zz.org&gt;" may not be updated.
+	 WARNING     "Your Name &lt;yy@zz.org&gt;" qsTr may not be used
```

Comment:
there are a lot of false positive for this warning.
We can try to simplify it later or at least add a list of false positive to skip

Reply:
yes, this is what i was concerned about too. Some other way has to be found for checking qsTr to avoid these false positives.